src/sentry/static/sentry/app/components/stream/group.tsx

```diff
@@ -395,7 +395,9 @@ class StreamGroup extends React.Component<Props, State> {
             showInboxTime={showInboxTime}
           />
         </GroupSummary>
-        {hasGuideAnchor && <GuideAnchor target="issue_stream" />}
+        {hasGuideAnchor && (
+          <GuideAnchor target="issue_stream" disabled={!hasGuideAnchor} />
```

Comment:
shouldn't need disabled here since the guide only gets rendered when hasGuideAnchor is true